Sources/TuistCore/Utils/GraphLoaderCache.swift

```diff
@@ -0,0 +1,88 @@
+import Basic
+import Foundation
+
+/// Graph loader cache.
+public class GraphLoaderCache: GraphLoaderCaching {
+    /// Initializer
+    public init() {}
+
+    // MARK: - GraphLoaderCaching
+
+    var tuistConfigs: [AbsolutePath: TuistConfig] = [:]
```

Comment:
can we restrict public mutation on these properties as it appears as if all these properties have respective functions through which mutation occurs, change is not warranted in the scope of this PR though?

Reply:
> can we restrict public mutation on these properties as it appears as if all these properties have respective functions through which mutation occurs, change is not warranted in the scope of this PR though?

Yeah, we can restrict the mutation but I wouldn't do it as part of this PR? Would you like to do it in a different PR? Otherwise we can create an issue as an improvement to add in the future.